resources/public/api/conf/1.0/application.raml

```diff
@@ -83,7 +83,7 @@ uses:
         uriParameters:
           taxYear:
             description: 'The tax year the data applies to, for example, 2021-22. The tax year must have ended. The start year and end year must not span two tax years. The minimum tax year is 2021-22. No gaps are allowed, for example, 2020-22 is not valid. (The minimum tax year in Sandbox is 2019-20.)'
-            example: "2019-20"
+            example: "2021-22"
```

Comment:
Pensions isn't part of this change - and in the code, there's nothing specifying the minimum year in sandbox vs other environments. Something to discuss after stand up tomorrow?

Reply:
from memory its already in pentions, and its one of caseys endpoints not nicks and nick said he was going to talk to casey about when he was going to do any alignments required on that when hes back from his A/L